### DISCUSSION TITLE
Rulers Color

### DISCUSSION THREAD

**Discussion by babakabdolahi** [helix-editor/helix#3188] [General] [answered]
Hey guys, 
Is there any way to change the color of the rulers?

**Comment by kirawi** [comment, marked as answer]
You would have to change the color in your theme, e.g. https://github.com/helix-editor/helix/blob/dad6d0f3b2244f2f7d5b4ab8768bbdd11bc4dd0a/runtime/themes/dark_plus.toml#L83

The theme files should be contained in [your runtime folder.](https://github.com/helix-editor/helix/wiki/FAQ#access-the-helix-config-directory)

**Reply by babakabdolahi** [reply]
Thanks @kirawi
